latex_fill_all.py

```diff
@@ -874,22 +874,46 @@ class LatexFillAllCommand(
         insert an entry; if force is true, completion_type must be a string;
         if force is true, the bracket matching and word overwriting behaviour
         is disabled
+
+    :param single_selection:
+        positive integer indicating to ignore multiple cursors/carets on the
+        text and use one specific selection point. This is useful when creating
+        a snippet which defaults to creating multiple carets/cursors and want to
+        force the completion for some specific scope as `input` or `cite`.
+        Therefore we must to ignore the multi selection, otherwise it will
+        ignore the input scope specified
+        the accepted values start on 0 for the first selection, 1 for the second
+        and etc. They must to be lower then the maximum selections, otherwise
+        this setting will be ignored.
     '''
 
     def is_visible(self, *args):
         return bool(self.view.score_selector(0, "text.tex.latex"))
 
     def run(
         self, edit, completion_type=None, insert_char="", overwrite=False,
-        force=False
+        force=False, single_selection=-1
```

Comment:
I would prefer `single_selection=None` (and `single_selection is not None and ...` later)